xcell/mappers/mapper_DESY1wl.py

```diff
@@ -108,10 +117,43 @@ def _set_mode(self, mode=None):
             raise ValueError(f"Unknown mode {mode}")
         return e1_flag, e2_flag, mode
 
+    def _get_Rs():
```

Comment:
I'd check that Rs is not already set. Otherwise, if it is somehow called, it will fail as you will have had clean the catalog.